pulp_ansible/app/viewsets.py

```diff
@@ -373,13 +693,51 @@ def sign(self, request, pk):
         return OperationPostponedResponse(result, request)
 
 
-class AnsibleRepositoryVersionViewSet(RepositoryVersionViewSet):
+class AnsibleRepositoryVersionViewSet(RepositoryVersionViewSet, RolesMixin):
```

Comment:
Remove the `RolesMixin` and its actions. We don't assign roles/perms to repository versions, it's all based off the parent repository.